fetch/metadata/preload.https.sub.html

```diff
@@ -22,7 +22,7 @@
       if (as !== undefined) {
         e.setAttribute("as", as);
       }
-      e.onload = e.onerror = t.step_func_done(e => {
+      e.onload = e.onerror = t.step_func(e => {
         fetch("/fetch/metadata/resources/record-header.py?retrieve=true&file=" + key)
           .then(t.step_func(response => response.text()))
           .then(t.step_func(text => assert_header_equals(text, expected)))
```

Comment:
Though note that `resolve()` on the line below is undefined, and there's some duplicate subtests names from who-knows-where:
```
 0:30.57 INFO |                   Test                   |           Subtest           |          Results           | Messages |
 0:30.57 INFO |------------------------------------------|-----------------------------|----------------------------|----------|
 0:30.57 INFO | `/fetch/metadata/preload.https.sub.html` | `undefined: sec-fetch-dest` | **Duplicate subtest name** |          |
 0:30.57 INFO | `/fetch/metadata/preload.https.sub.html` | `undefined: sec-fetch-mode` | **Duplicate subtest name** |          |
 0:30.57 INFO | `/fetch/metadata/preload.https.sub.html` | `undefined: sec-fetch-site` | **Duplicate subtest name** |          |
 0:30.57 INFO | `/fetch/metadata/preload.https.sub.html` | `undefined: sec-fetch-user` | **Duplicate subtest name** |          |
```

Reply:
Turns out, when you don't actually run the test code, it sometimes doesn't work. :)

I'll take another pass, thank you.